ci/authn-k8s/test_gke_entrypoint.sh

```diff
@@ -29,6 +29,14 @@ function finish {
       echo "Logs from Conjur Pod $pod_name:"
       kubectl logs $pod_name > output/gke-authn-k8s-logs.txt
 
+      # Rails.logger writes the logs to the environment log file
+      kubectl exec $pod_name -- bash -c "cat /opt/conjur-server/log/test.log" >> output/gke-authn-k8s-logs.txt
```

Comment:
Do you want a separator to show that there are two log files being combined here? Some kind of header may be useful.

Reply:
it doesn't really matter